src/main/webapp/3Dscript.js

```diff
@@ -1,31 +1,49 @@
-// import {FlyControls} from 'https://threejs.org/examples/jsm/controls/FlyControls.js';
 import {OrbitControls} from 'https://threejs.org/examples/jsm/controls/OrbitControls.js';
 import {GUI} from 'https://threejs.org/examples/jsm/libs/dat.gui.module.js';
-
+/**
+* A Point object with a lat and lng.
+* @typedef {Object<number, number, number>} Point
+* @property {number} lat The lat of the point.
+* @property {number} lng The lng of the point.
+* @property {number} alt The alt of the point.
+*/
 // These are global objects.
+/**
+ * @typedef {Array<number, number>}
```

Comment:
What does it mean for an Array to have multiple parameters? And does this mean `runs` is an array, but also has the properties `data` and `color`?

Reply:
It should be Array<Object<number,number>>. Will resolve this

Reply:
Try to give a descriptive name to make your code have better readability.
E.g. runIdToData.